Update Software docs with Argo info from Astronomer forum (#1300)
astronomer/docs: software/install-aws-standard.md

After you run the previous commands, a set of Kubernetes pods are generated in your namespace. These pods power the individual services required to run the Astronomer platform, including the Software UI and Houston API.

:::info Alternative ArgoCD installation

> You can install Astronomer with ArgoCD as an alternative to using `helm install`. Because ArgoCD doesn't support sync wave dependencies for [app of apps](https://argo-cd.readthedocs.io/en/stable/operator-manual/cluster-bootstrapping/#app-of-apps-pattern) structures, this process requires some additional steps compared to the standard ArgoCD workflow:

Reviewer: You can install Astronomer with ArgoCD as an alternative to using `helm install`. 

Why would I want to do this? What benefit does it provide? Is it easier? Does it take less time?

Reviewer: I wasn't asking that you "elaborate on the specific benefits of ArgoCD". I thought a small value message statement might help users understand the benefits of using ArgoCD over helm. If you don't think it's necessary, no worries.

> You can install Astronomer with ArgoCD as an alternative to using `helm install`. Because ArgoCD doesn't support sync wave dependencies for [app of apps](https://argo-cd.readthedocs.io/en/stable/operator-manual/cluster-bootstrapping/#app-of-apps-pattern) structures, this process requires some additional steps compared to the standard ArgoCD workflow:

Reviewer: ```suggestion
ArgoCD doesn't support sync wave dependencies for App of Apps patterns and you need to complete this process to install Astronomer Software. For more information about App of Apps patterns, see [App of Apps Pattern](https://argo-cd.readthedocs.io/en/stable/operator-manual/cluster-bootstrapping/#app-of-apps-pattern).
```

Author: Not sure this is quite what I was trying to say in the first sentence. I was trying to communicate "you might think that this will be just like all of your other ArgoCD installs, but Astronomer is weird so you have to do the following custom setup"

Reviewer: Fair enough. I struggled with the original wording. Probably just my unfamiliarity with subject matter. Also, I was trying to eliminate the colon. This is why I try and avoid blocking PRs - writing is subjective.  :)

1. Add `enableArgoCDAnnotation: true` to your `config.yaml` file.
2. Create a new ArgoCD app. When creating the app, configure the following:

>     - **Path**: The filepath of your `config.yaml` file

Author: @rbankston Are there any other values people should set here?

Reviewer 2: just the normal values like base domain and any configurations they need for their install

    - **Namespace**: `astronomer`
    - **Cluster**: The Kubernetes cluster in which you're installing Astronomer 

>     - **Repository URL**: `https://github.com/astronomer/astronomer`

Reviewer 2: have always used https://helm.astronomer.io if installing as a helm app

> 3. Sync the application with every component of the Astronomer platform selected. Once you see `astronomer-houston-db-migrations`, terminate the sync.

Reviewer: ```suggestion
3. Sync the ArgoCD app with every component of the Astronomer platform selected. Once you see `astronomer-houston-db-migrations`, terminate the sync.
```

You've used _app_ previously, so you should use it in the following procedure steps for consistency.

> 4. Sync the application a second time, but this time unselect `astronomer-alertmanager` while keeping all other components selected. Wait for this sync to finish completely.

Reviewer: ```suggestion
4. Sync the application a second time, but clear `astronomer-alertmanager` while keeping all other components selected. Wait for this sync to finish completely.
```
If it's a checkbox, you use _clear_ and and not _unselect_.

Alternative:

Sync the ArgoCD app a second time with all components but `astronomer-alertmanager` selected. Wait for the sync to finish before proceeding to step 5.

> 5. Sync the application a third time with every component of the Astronomer platform selected. 

Reviewer: ```suggestion
5. Sync the ArgoCD app a third time with all Astronomer platform components selected. 
```

After you run the previous commands, a set of Kubernetes pods are generated in your namespace. These pods power the individual services required to run the Astronomer platform, including the Software UI and Houston API.

### Alternative ArgoCD installation

You can install Astronomer with [ArgoCD](https://argo-cd.readthedocs.io/en/stable/), which is an open source continuous delivery tool for Kubernetes, as an alternative to using `helm install`. 

> Because ArgoCD doesn't support sync wave dependencies for [app of apps](https://argo-cd.readthedocs.io/en/stable/operator-manual/cluster-bootstrapping/#app-of-apps-pattern) structures, installing Astrnonomer requires some additional steps compared to the standard ArgoCD workflow:

Author: ```suggestion
Because ArgoCD doesn't support sync wave dependencies for [app of apps](https://argo-cd.readthedocs.io/en/stable/operator-manual/cluster-bootstrapping/#app-of-apps-pattern) structures, installing Astronomer requires some additional steps compared to the standard ArgoCD workflow:
```
